android/engine/src/main/java/org/smartregister/fhircore/engine/rulesengine/DateTimeHelper.java

```diff
@@ -0,0 +1,22 @@
+package org.smartregister.fhircore.engine.rulesengine;
+
+// Online Java Compiler
+// Use this editor to write, compile and run your Java code online
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateTimeHelper {
+
+    public static String makeItReadable(String dateString) {
+        SimpleDateFormat from = new SimpleDateFormat("yyyy-MM-dd");
+        Date date = null;
+        try {
+            date = from.parse(dateString);
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        return new SimpleDateFormat("E, MMM dd yyyy").format(date);
+    }
+}
```

Comment:
This implementation is actually not necessary. Use available implementation from the `DateTimeExtension`. If possible refactor the `extractDOB` function of the RulesEngineService to be generic. It has a parameter for providing the data format.

Reply:
Since extractDOB has a usage so I have added additional method to support prettyTime library in rules, I've added couple of variants for accessibility according to needs. We can mark extractDOB as deprecate. Please check and give feedback. Thanks